scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -822,7 +822,7 @@
               matched-considerable-jobs-head?)))
         (catch Throwable t
           (meters/mark! handle-resource-offer!-errors)
-          (log/error t "In" pool-name "pool, error in match:" (ex-data t))
+          (log/info t "In" pool-name "pool, error in match:" (ex-data t))
```

Comment:
Can we change to INFO for only the "The job can't start now" exception?

Seems like other exceptions should still be ERROR.

Reply:
There are no other exception subtypes in the last 30 days thrown. This code is not executed in our integration tests. I'm reluctant to make any change more invasive than this without testing it, and testing it pushes this change over the quicky level of effort.

Reply:
We control the "The job can't start now" exception and we catch it here. It seems easy to add a check for that exception (e.g. with a field in the exception info map) and only log `info` for that case.